crates/astria-sequencer-relayer/Cargo.toml

```diff
@@ -29,7 +29,10 @@ tokio = { workspace = true, features = [ "macros", "rt-multi-thread" ] }
 astria-gossipnet = { path = "../astria-gossipnet" }
 astria-proto = { path = "../astria-proto" }
 astria-rs-cnc = { path = "../astria-rs-cnc" }
+astria-sequencer-client = { path = "../astria-sequencer-client" }
+
 base64-serde = "0.7.0"
+home = "0.5.5"
```

Comment:
Don't use this in a service.

Reply:
i have no idea why but changing the validator key path seems to have made the default file un-findable for me :/ will remove though

Reply:
Yeah, the default place to look for cometbft is a bit iffy. It makes sense if you run this as a user, but it's different for services.

In my opinion we should remove all defaults in these configs and provide all of it through env vars. That will a) cut down on complexity and b) means we have one source of truth (which is nice)